blueprints/addon/additional-package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Seemed easier to move more stuff into an "additional" package.json file, instead of adding everything in JS land.